### ISSUE TITLE
[Bug]在服务器死亡后能力会被假重置，退出重进服务器才能恢复

### ISSUE THREAD

**Issue by Drogenx** [LambdaInnovation/AcademyCraft#801]
**Describe the bug**
在服务器死亡后能力会被假重置，退出重进服务器才能恢复

**To Reproduce**
1. 死亡
2. 能力回到初始状态

**Expected behavior**

**Screenshots**
![2019-06-19_23 22 59](https://user-images.githubusercontent.com/49275165/59778772-ad0f3100-92e9-11e9-96bd-d3b14039df6a.png)
![2019-06-19_23 23 09](https://user-images.githubusercontent.com/49275165/59778774-ae405e00-92e9-11e9-9156-dc9dac668a28.png)
![2019-06-19_23 23 27](https://user-images.githubusercontent.com/49275165/59778778-aed8f480-92e9-11e9-934e-e44513ef52ed.png)

**Desktop (please complete the following information):**
 - OS: [Windows10]
 - MC Version and Forge version: [14.23.5.2838-1.12.2]
 - Version [1.1.2]

**Additional context**

**Comment by Ruchian** [comment]
Tested the following:
Using: Forge 14.32.5.2838-1.12.2, AcademyCraft 1.1.2, Just Enough Items 1.12.2-4.15.0.281

I set up a quick flat world with a windmill, ability developer, node and imag matrix, connected them all to power the developer. (played in survival mode, singleplayer)
Upon >learn, I obtained the Vector Manip ability category.

I used the command /kill, to commit suicide.

What I noticed was that I still had the ability. Like the original poster, I tried learning the first skill and then afterwards use /kill. Nothing changed. In other words, I cannot confirm Drogenx problem.

Upon closer inspection of his screenshots I realized that he is playing on a modded server that allows plugins. (likely CatServer or something.) The /suicide command doesn't exist in native minecraft, nor the mod. Perhaps his problems happen due to a plugin's effect.
